boot/zephyr/main.c

```diff
@@ -73,7 +81,7 @@ void main(void)
 	SYS_LOG_INF("Setting vector table to %p", vt);
 
 	/* Not all targets set the VTOR, so just set it. */
-	_scs_relocate_vector_table((void *) vt);
+	relocate_vector_table((void *) vt);
```

Comment:
Have we determined we want to set the vector table in the bootloader at all? It should be fixed for all Cortex-M targets in Zephyr, and I would argue if any are missing, it should be fixed there, and this code just removed entirely from here.